include/seqan3/alphabet/cigar/all.hpp

```diff
@@ -0,0 +1,41 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \author Joshua Kim <joshua.kim AT fu-berlin.de>
+ * \brief Meta-header for the cigar submodule; includes all headers from alphabet/cigar/.
+ */
+
+ #pragma once
+
+ #include <seqan3/alphabet/cigar/cigar_op.hpp>
+
+ /*!\defgroup cigar CIGAR
+  * \brief Contains the CIGAR operation alphabet, along with the CIGAR cartesian composition.
+  * \ingroup alphabet
+  *
+  * \par Introduction
+  *
+  * CIGAR strings are combinations of count values and CIGAR operations. This submodule has two different
```

Comment:
Please mention in the introduction that a CIGAR string represents an **alignment** as a sequence of edit operations.